modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1268,6 +1268,58 @@ definition {
 		}
 	}
 
+	@description = "LPS-165994 - Verify that the user cannot insert an invalid date on Start / End fields when creating a filter"
```

Comment:
You could use the LPS from the story ticket here: `LPS-156704`

Reply:
Change the `LPS-165994` (ticket number) to `LPS-156704`

Reply:
Changed the LPS in the description.